vantage6/vantage6/cli/configuration_wizard.py

```diff
@@ -182,6 +183,77 @@ def node_configuration_questionaire(dirs: dict, instance_name: str) -> dict:
     return config
 
 
+def _get_allowed_algorithms() -> list[str]:
+    """
+    Prompt the user for the allowed algorithms on their node
+
+    Returns
+    -------
+    list[str]
+        List of allowed algorithms or regular expressions to match them
+    """
+    info("Below you can add algorithms that are allowed to run on your node.")
+    info(
+        "You can use regular expressions to match multiple algorithms, or you can "
+        "use strings to provide one algorithm at a time."
+    )
+    info("Examples:")
+    # pylint: disable=W1401
+    # flake8: noqa: W605
+    info("^harbor2\.vantage6\.ai/demo/average$    Allow the demo average algorithm")
+    info(
+        "^harbor2\.vantage6\.ai/algorithms/.*   Allow all algorithms from "
+        "harbor2.vantage6.ai/algorithms"
+    )
+    info(
+        "^harbor2\.vantage6\.ai/demo/average:@sha256:82becede...$    Allow a "
+        "specific hash of average algorithm"
+    )
+    allowed_algorithms = []
+    while True:
+        algo = q.text(message="Enter your algorithm expression:").ask()
+        allowed_algorithms.append(algo)
+        if not q.confirm(
+            "Do you want to add another algorithm expression?", default=True
+        ).ask():
+            break
+    return allowed_algorithms
+
+
+def _get_allowed_algorithm_stores() -> list[str]:
+    """
+    Prompt the user for the allowed algorithm stores on their node
+
+    Returns
+    -------
+    list[str]
+        List of allowed algorithm stores
+    """
+    info("Below you can add algorithm stores that are allowed to run on your node.")
+    info(
+        "You can use regular expressions to match multiple algorithm stores, or you can"
+        " use strings to provide one algorithm store at a time."
+    )
+    info("Examples:")
+    info(
+        "https://store.cotopaxi.vantage6.ai    Allow all algorithms from the "
+        "community store"
+    )
+    info(
+        "^https://*\.vantage6\.ai$               Allow all algorithms from any "
```

Comment:
I guess here we are also missing a dot or something (e.g. `[a-z]`) before the Kleene star..


Or, something like..
```suggestion
        "^https://[a-z]+\.vantage6\.ai$               Allow all algorithms from any "
```